repose-aggregator/tests/functional-tests/src/integrationTest/groovy/framework/mocks/MockIdentityV2Service.groovy

```diff
@@ -1054,7 +1054,11 @@ class MockIdentityV2Service {
             "tenant":{
                 "id": "\${tenantid}",
                 "name": "\${tenantname}"
-            }
+            },
+            "RAX-AUTH:authenticatedBy": [
+                "PASSWORD",
+                "MFA"
```

Comment:
You may want to change "MFA" to "PASSCODE" so that we don't think "MFA" is a valid value in the future.